### DISCUSSION TITLE
From exchange to chia app wallet balance does not shows

### DISCUSSION THREAD

**Discussion by nottynash89** [Chia-Network/chia-blockchain#4473] [Support]
Hello, I bought some chia from an exchange and transferred it to my chia app wallet but it never shows on my wallet, my app full node is synced and updated to latest 1.1.15, when i look up the block chain explorer it shows the transaction there, but nothing came in the app wallet?
can this wallet receive xch from outside or it can only be used for faming?

**Comment by ilyakooo0** [comment]
1.1.15 does not currently exist. Do you mean 1.1.5?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
